_posts/2018-11-20-mongodb-europe-018.md

```diff
@@ -0,0 +1,411 @@
+
+
+# [MongoDB Enterprise Operator for Kubernetes at Amadeus](https://sched.co/FmAc){:target="_blank" rel="noopener noreferrer"}
+
+[Amadeus](https://amadeus.com/en) is the world's largest technology company dedicated to the travel industry.
+They have developed an impressive MongoDB farm, a large environment with 100 clusters.
+Some of these clusters run more than 100 shards, while others have 100TB MongoDB databases.
+Amadeus processes more than 1 trillion flight availability requests per day. 
+For each single search you do on a website they receive 500.000 availability requests.
+So search responsibly ;-)
+The number of requests per day grows by 50% each year.
+The second of January has by far the most requests, due to new years resolutions!
+If this day is in the weekend all systems are pushed to their limits.
+The fair database for one of their big clients, Kayak, is 100TB in size and changes daily.
+That's some pretty big numbers there.
+No wonder that Amadeus is a happy user of the MongoDB Enterprise Operator for Kubernetes.
+
+Starting with the MongoDB Ops Manager v4.0, MongoDB officially supports the management and deployment of MongoDB in Kubernetes with Backup, Automation, Alerting and Monitoring.
+A [MongoDB Enterprise Kubernetes Operator](https://github.com/mongodb/mongodb-enterprise-kubernetes) has app-specific awareness about stateful applications, so it knows how to deploy them.
+This operator helps automating scripted tasks and enables MongoDB-as-a-service for developers.
+It talks to Ops Manager and delegates the creation of clusters, shards, backups and automation to Ops Manager.
+The underlying necessary Kubernetes infrastructure is orchestrated by the operator itself and so they work in conjunction.
+This provides for clusters to be setup, scaled up/down/out/in, with a single simple yaml file.
+And kubernetes provides the self-healing capabilities, how nice is that!?
+
+The following yaml file is all you need to spin up a 3 node replica set :
+
+```yaml
+apiVersion: mongodb.com/v1
+kind: MongoDbReplicaSet
+metadata:
+  name: myReplicaSet
+  namespace: mongodb
+spec:
+  members: 3
+  version: 4.0.1
+
+  project: projectName
+  credentials: myUberSecretCredentials
+```
+
+I kid you not, that's it.
+
+Scale out or back in with a simple change in the config yaml and 
+```bash
+cubectl apply -f file.yaml
+```
+
+# MongoDB Charts
+- [Bringing Data to Life with MongoDB Charts](https://sched.co/FtWP), Guillaume Meister.
+
+Currently, if you want to visualize data in a MongoDB database you either have to code it yourself, or rely on a 3rd party tool and migrate your data to a different platform (for example: Kibana with Elasticsearch is a popular tool).
+Needless to say that this can be quite cumbersome.
+[MongoDB Charts](https://www.mongodb.com/products/charts) intends to solve this.
+
+So what is it? 
+MongoDB Charts is a visualization tool that runs as a standalone webserver, you access it via a web browser.
```

Comment:
Isn't it rather redundant that you can access a standalone web server through a web browser?